doc/changelog.rst

```diff
@@ -6,6 +6,85 @@ Change Log
 
 .. towncrier release notes start
 
+QuTiP 5.0.0a2 (2023-09-06)
+==========================
+
+Features
+--------
+
+- Add support for different spectra types for bloch_redfield_tensor (#1951)
+- Improve qutip import times by setting logger names explicitly. (#1981, by Pieter Eendebak)
+- Change the order of parameters in expand_operator (#1991)
+- Add `svn` and `solve` to dispatched (#2002)
+- Added nm_mcsolve to provide support for Monte-Carlo simulations of master equations with possibly negative rates. The method implemented here is described in arXiv:2209.08958 [quant-ph]. (#2070 by pmenczel)
+- Add support for combining bosinic and fermionic HEOM baths (#2089)
+- Added __repr__ to QobjEvo (#2111 by lklivingstone)
+- Improve print(qutip.settings) by make it shorter (#2113 by tamakoshi2001)
+- Create the `trace_oper_ket` operation (#2126)
+- Speed up the construction of the RHS of the HEOM solver by a factor of 4x by converting the final step to Cython. (#2128)
+- Rewrite the stochastic solver to use the v5 solver interface. (#2131)
+- Add `Qobj.get` to extract underlying data in original format. (#2141)
+- Add qeye_like and qzero_like (#2153)
+- Add capacity to dispatch on ``Data`` (#2157)
+- added fermionic annihilation and creation operators.
+  Closely followed the protocol proposed in the following `source
+  <https://github.com/qutip/qutip/issues/863>`_. (#2166 by khnikhil)
```

Comment:
```suggestion
- Added fermionic annihilation and creation operators. (#2166 by khnikhil)
```
I found the wording of the link to the issue a bit unhelpful (the issue has lots of back and forth discussion about how to correctly do this and not a clear description of what was implemented).